src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -746,4 +746,12 @@
     <value>Suggestions found: {0}</value>
     <comment>{0} will be replaced with a number.</comment>
   </data>
-</root>
+  <data name="NewTabAnnouncement" xml:space="preserve">
+    <value>A new tab for the "{}" profile was created successfully</value>
+    <comment>{} will be replaced with the name of the profile (i.e. PowerShell, Command Prompt, etc.). The name may span multiple words. This announcement is read by a screen reader.</comment>
+  </data>
+  <data name="SplitPaneAnnouncement" xml:space="preserve">
+    <value>A new pane for the "{}" profile was created successfully</value>

```

Comment:
A11y team requested: "profile name created successfully"

We've diverged a bit since then haha. I find value in explicitly saying "pane" or "tab". I also think saying "profile" is important because some profile names could be weird. That said, what if we change it to...
`"{}" pane/tab created successfully`

It's more succinct? But there's still some doubts when it comes to [localization](https://github.com/microsoft/terminal/pull/13575#discussion_r928040951). Thoughts?